sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/MSIToken.java

```diff
@@ -102,4 +106,29 @@ private static Long parseToEpochSeconds(String expiresOn, String expiresIn) {
         }
         throw LOGGER.logExceptionAsError(new IllegalArgumentException("Unable to parse date time " + dateToParse));
     }
+
+    private static OffsetDateTime parseRefreshIn(String refreshIn) {
```

Comment:
This seems unused?

Reply:
yeah, this is removed, looks like the commit isn't pushed.